sktime/forecasting/naive.py

```diff
@@ -237,3 +237,29 @@ def _predict_last_window(
                     # linear extrapolation
                     y_pred = last_window[-1] + (fh_idx + 1) * slope
                     return y_pred
+
+    def compute_pred_int(self, y_pred, alpha=DEFAULT_ALPHA):
+        fh = list(range(1, len(y_pred)+1))
+        data = np.array(self._y)
+        T = len(data)
+        K = 1
+
+        actual = data[0 : T - 1]
+        predicted = data[1:T]
+
+        errors = (actual - predicted) ** 2
```

Comment:
Here this error calculation is correct to the naive("last") method but I don't think it is correct to other methods("mean" and "drift"). So I still need to calculate them. And calculate sigma_hat separately for each case after calculating the errors. I'm planning to do it inside the if conditions(line 254-261). Do you have any other suggestions?

Reply:
Or should I calculate them inside a new different function inside the naive.py?

Reply:
It might be safer to create a completely new forecaster instead of using self.predict(), since in my suggestion you'd be overwriting the forecasting horizon of self, which is probably not a good idea!